packages/core/src/bag3d/core/assets/ahn/download.py

```diff
@@ -393,7 +427,12 @@ def match_sha(
     if not fpath.is_file():  # pragma: no cover
         raise FileNotFoundError(fpath)
     sha = sha_func.compute(fpath)
-    if sha.hexdigest() == sha_reference[fpath.name]:
+    if not sha_reference[fpath.name]:
```

Comment:
Here, if the checksum is not available the function still return True. This is for still being able to download AHN5 tiles which are available in the tile_index but we do not have in the checksum list yet.